Library/Formula/zanata-cli.rb

```diff
@@ -0,0 +1,19 @@
+class ZanataCli < Formula
+  desc "Zanata translation system command-line client"
+  homepage "http://zanata.org/"
+  url "http://search.maven.org/remotecontent?filepath=org/zanata/zanata-cli/3.7.4/zanata-cli-3.7.4-dist.tar.gz"
+  sha256 "4424322bffb81f5185f87d1f11c7e92cde504c736695dbb4ced5854e49672037"
+
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/bin/zanata-cli"
+    bash_completion.install "#{libexec}/bin/zanata-cli-completion"
+  end
+
+  test do
+    %x(JAVA_HOME=\`/usr/libexec/java_home\` zanata-cli --help) =~
```

Comment:
If you need to set `JAVA_HOME` here do it with `ENV["JAVA_HOME"]` rather than this way, but it may not be necessary. For the actual check you can do:

```ruby
assert_match /Zanata Java command-line client/, shell_output("#{bin}/zanata-cli --help")
```